core_v2/sources/v2_domains.move

```diff
@@ -108,23 +111,27 @@ module aptos_names_v2::v2_domains {
         token_addr: Option<address>,
     }
 
+    #[resource_group_member(group = aptos_names_v2::v2_domains::ObjectGroup)]
     /// Holder for `SetReverseLookupEvent` events
-    struct SetReverseLookupEvents has key, store {
+    struct SetReverseLookupEvents has key {
         set_reverse_lookup_events: event::EventHandle<SetReverseLookupEvent>,
     }
 
+    #[resource_group_member(group = aptos_names_v2::v2_domains::ObjectGroup)]
     /// Holder for `SetTargetAddressEvent` events
-    struct SetTargetAddressEvents has key, store {
+    struct SetTargetAddressEvents has key {
         set_name_events: event::EventHandle<SetTargetAddressEvent>,
```

Comment:
Why separate events into so many structs?

Reply:
If most of these functions that emit different events change NameRecord, the conflict would still happen anyway right?